tests/config/test_document_type_config.py

```diff
@@ -160,3 +163,80 @@ def test_validate_document_invalid_extension(invalid_extension, valid_document):
         match=r"The extension provided for document type .* is not alphanumeric.",
     ):
         validate_document_type_config("ext", valid_document)
+
+
+@pytest.mark.skipif(sys.platform != "darwin", reason="Test runs only on macOS")
+def test_document_type_macOS_config_with_mimetype_single(valid_document):
+    """Valid document types don't raise an exception when validated.
+
+    application/pdf is the only valid MIME type for PDF files.
+    """
+    valid_document["mime_type"] = "application/pdf"
+    validate_document_type_config("ext", valid_document)
+    assert "LSItemContentTypes" in valid_document["macOS"].keys()
+    assert valid_document["macOS"]["LSItemContentTypes"] == "com.adobe.pdf"
```

Comment:
We should be checking `is_core_type`, `LSHandlerRank`, and `UTTypeConformsTo` in this and subsequent tests, just to confirm we're getting all the right keys.

Reply:
Done